src/components/Footer.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import './style.css';
+
+function Footer() {
+  return (
+    <div className='main-footer'>
```

Comment:
this change seems unrelated

Reply:
Sir this was not done by us ,while managing conflicts we accepted the changes.

Reply:
Rebase your branch then. Thanks